Students/Truta Andrei/Lab04/src/ro/utcn/Customer.java

```diff
@@ -0,0 +1,87 @@
+package ro.utcn;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Customer {
+    private Integer id;
+    private String name;
+    private String address;
+    private String email;
+    private String phone;
+    private List<Order> placedOrders = new ArrayList<Order>();
+
+    static Integer lastId = -1;
+
+    static Integer getNextId() {
```

Comment:
asta puteai inclusiv sa o ascunzi catre exteriorul clasei si sa o folosesti in cadrul constructorului